### PR TITLE
Create initial MFRC522 documentation

### PR DESCRIPTION
**Pull request by UdoWeberJR**
Linked the data sheet and created an example for a Raspi 3B+ with SPI.
Added hint for bad power supply.

### CODE REVIEW

File: site/content/device/MF-RC522 [hunk 1]
```diff
@@ -0,0 +1,185 @@
++++
+title = "MF-RC522"
+description = "1 bit OLED display controller"
```
Comment (maruel) [inline site/content/device/MF-RC522]: Oops?

**Review by maruel — COMMENTED** [review]
You should be able to git commit an image file and update the links to it as everything in /static/ is available at the root, e.g. https://periph.io/img/fan.jpg.

**Comment by UdoWeberJR** [comment]
Changed it.

File: site/content/device/MF-RC522 [hunk 2]
```diff
@@ -1,7 +1,7 @@
 +++
 title = "MF-RC522"
-description = "1 bit OLED display controller"
-picture = "/img/ssd1306.jpg"
+description = "MIFARE and NTAG frontend"
+picture = "/img/mfrc522.JPG"
```
Comment (maruel) [inline site/content/device/MF-RC522]: Did you mean to git add the picture?

File: site/content/device/MF-RC522 [hunk 3]
```diff
@@ -1,7 +1,7 @@
 +++
```
Comment (maruel) [inline site/content/device/MF-RC522]: Please name the file `mf-rc522.md` as all the other files are lower case, and it won't work without a .md file extension. Thanks!

File: site/content/device/MF-RC522 (hunk 2 above)
Comment (UdoWeberJR) [reply site/content/device/MF-RC522]: I do not know, I just uploaded the picture in the same folder as the others. I never used git hub before and have no clue.

File: site/content/device/MF-RC522 (hunk 3 above)
Comment (UdoWeberJR) [reply site/content/device/MF-RC522]: Can not find an option in this Webinterface to change the name of the file?

Comment (UdoWeberJR) [reply site/content/device/MF-RC522]: Ok, changed the name.

File: site/content/device/MF-RC522 (hunk 2 above)
Comment (maruel) [reply site/content/device/MF-RC522]: No worry, the only thing you need to do is `git add` with the path of the image, the `git commit` then `git push`.

**Pull request closed without merging**